samples/DataImporter/GraphEngine.DataImporter/DsvImporter.cs

```diff
@@ -8,15 +8,21 @@
 
 namespace GraphEngine.DataImporter
 {
-    class CsvImporter : IImporter<string>
+    class DsvImporter : IImporter<string>
     {
         private List<string> m_fieldNames;
+        private char [] separators;
+
         private JsonImporter m_jsonImporter = new JsonImporter();
 
+        public DsvImporter(char [] FileSeparators)
```

Comment:
>char [] FileSeparators [](start = 27, length = 22)

1) Parameters usually start with lowercase letters.
2) delimiters is a better name?
3) why char[], not char?